snapcraft/main.py

```diff
@@ -35,13 +35,15 @@
   snapcraft [options] keys
   snapcraft [options] register-key [<key-name>]
   snapcraft [options] register <snap-name> [--private]
-  snapcraft [options] sign-build <snap-file> [--key-name <key-name>] [--local]
+  snapcraft [options] sign-build <snap-file> [--key-name=<key-name>] [--local]
   snapcraft [options] upload <snap-file>
   snapcraft [options] push <snap-file> [--release <channels>]
   snapcraft [options] release <snap-name> <revision> <channel>
   snapcraft [options] list-plugins
   snapcraft [options] tour [<directory>]
   snapcraft [options] update
+  snapcraft [options] gated <snap-name>
+  snapcraft [options] validate <snap-name> [<snap-revision> ...] [--key-name=<key-name>]
```

Comment:
@sergiusens I just pushed a change where it now is <validation>... which renames it, and makes it required at least once.